src/Medianizer.sol

```diff
@@ -72,6 +78,7 @@ contract Medianizer {
             median = data[(count - 1) / 2];
         }
 
+        if (tags[tag] != bytes32(0)) tag = tags[tag];
```

Comment:
same thing, let's use source tags.  multiple dests per source vs multiple sources per dest, the idea is to poke the dest

Reply:
yeah also here, we want to have (source, tag)[] => tag.  so maybe one medianized value can be from receiver, one can be from divider, etc